pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>me.lokka30</groupId>
     <artifactId>treasury-parent</artifactId>
-    <version>1.0.1-SNAPSHOT</version>
+    <version>1.1-SNAPSHOT</version>
```

Comment:
`1.1.0-SNAPSHOT`

Reply:
current 1.0.1-SNAPSHOT version is fine. yannick should revert that change

Reply:
Agreed.